modules/gitops-synchronizing-your-application-application-with-your-git-repository.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-your-application-application-with-your-git-repository_{context}"]
+= Synchronizing your application with your Git repository
+
+.Procedure
+. In the Argo CD dashboard, notice that the *cluster-configs* Argo CD application has the statuses *Missing* and *OutOfSync*. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically. 
+
+. Click the *Sync* button on the *cluster-configs* tile, review the changes, and then click *Synchronize*. Argo CD will detect any changes in the Git repository automatically. If the configurations are changed, Argo CD will change the status of the *cluster-configs* to *OutOfSync*. You can modify the synchronization policy for Argo CD to automatically apply changes from your Git repository to the cluster.
+
+. Notice that the *cluster-configs* Argo CD application now has the statuses *Healthy* and *Synced*. Click the *cluster-configs* tile to check the details of the synchronized resources and their status on the cluster.
+
+. Navigate to the OpenShift web console and click {rh-app-icon} and see that link to the *Red Hat Developer Blog* is now present there. 
```

Comment:
and click {rh-app-icon} to verify that a link to the *Red Hat Developer Blog* is now added.